lib/huginn_scheduler.rb

```diff
@@ -114,7 +114,7 @@ def run!
     end
 
     # Schedule event cleanup.
-    @rufus_scheduler.cron "0 0 * * * " + tzinfo_friendly_timezone do
+    @rufus_scheduler.every ENV['EVENT_EXPIRATION_CHECK'] do
```

Comment:
Important to set a default, so something like `ENV['EVENT_EXPIRATION_CHECK'].presence || '6h'`